app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    friends = [Person(someone["name"], someone["age"]) for someone in people]
+    for friend in friends:
```

Comment:
you can simplify the code by first creating all the Person objects, and then, in a subsequent loop, assigning the husbands and wives. At this stage, the husband or wife Person object will have already been created, facilitating this assignment.